src/routes/jira/jira-router.ts

```diff
@@ -6,6 +6,8 @@ import { JiraEventsRouter } from "./events/jira-events-router";
 import { JiraSelectVersionRouter } from "./events/jira-select-version-router";
 import { JiraContextJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
 import { JiraServerUrlRouter } from "./events/jira-server-url-router";
+import { JiraAppCreationRouter } from "./events/jira-app-creation-router";
+import { JiraVerifyServerUrlRouter } from "./events/jira-verify-server-url-router";
```

Comment:
These 2 files are in the wrong directory since they're not under the `/jira/events` routing path

Reply:
Moved them all into jira/server. That said, they don't all follow the file naming of jira-server-something-something. All files are related to the server installation flow. Happy to rename them/move them out into separate directories but will tackle that separately to minimize the number of changes up in here.